src/peliasDocGenerators.js

```diff
@@ -73,7 +73,11 @@ module.exports.create = function(hierarchy_finder) {
           wofDoc.addParent('macroregion', hierarchy_element.name, hierarchy_element.id.toString());
           break;
         case 'dependency':
-          wofDoc.addParent('dependency', hierarchy_element.name, hierarchy_element.id.toString());
+          if (hierarchy_element.abbreviation) {
```

Comment:
Someday we will refactor this function. Maybe it will take an improvement to the model code? (it's fine for now though)